Resources/Prototypes/Entities/Markers/Spawners/mobs.yml

```diff
@@ -922,4 +922,4 @@
       sprite: Mobs/Animals/reindeer_doe.rsi
   - type: ConditionalSpawner
     prototypes:
-    - MobReindeerDoe
\ No newline at end of file
+    - MobReindeerDoe
```

Comment:
Upstream file still shows as edited with only change being an EoF empty line added.
Please revert this change.